bridges/otellogr/logsink.go

```diff
@@ -10,13 +10,37 @@
 // way:
 //
 //   - Message is set as the Body using a [log.StringValue].
-//   - TODO: Level
+//   - Level is transformed and set as the Severity. The SeverityText is not
+//     set.
 //   - KeyAndValues are transformed and set as Attributes.
 //   - The [context.Context] value in KeyAndValues is propagated to OpenTelemetry
 //     log record. All non-nested [context.Context] values are ignored and not
 //     added as attributes. If there are multiple [context.Context] the last one
 //     is used.
 //
+// The V-level is transformed by using the [WithLevelSeverity] option. If this is
+// not provided it would default to a function that inverses the level with an
+// offset such that [logr.Info] is transformed to [log.SeverityInfo] as the higher
+// the V-level of a log line, the less critical it is considered. By default,
+// V-level higher than 8 is not supported and will result in negative severity.
+// For example:
+//
+//   - [logr.Info] is transformed to [log.SeverityInfo].
+//   - [logr.V(0)] is transformed to [log.SeverityInfo].
+//   - [logr.V(1)] is transformed to [log.SeverityDebug4].
+//   - [logr.V(2)] is transformed to [log.SeverityDebug3].
+//   - ...
+//   - [logr.V(8)] is transformed to [log.SeverityTrace].
+//   - [logr.V(9)] is transformed to [log.SeverityUndefined].
+//   - [logr.V(10)] is transformed to log.Severity(-1).
+//   - [logr.Error] is transformed to [log.SeverityError].
```

Comment:
After some thoughts I have a lot simpler proposal inspired by https://git-scm.com/docs/git-branch#Documentation/git-branch.txt--v

```suggestion
// The V-level is transformed by using the [WithLevelSeverity] option. If this is
// not provided there are transformed in the following way:
//
//   - logr.Info] and logr.V(0) are transformed to [log.SeverityInfo].
//   - logr.V(1) is transformed to [log.SeverityDebug].
//   - logr.V(2) and higher are transformed to [log.SeverityTrace].
```